Dockerfile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+FROM docker.io/eclipse-temurin:20-jdk AS builder
+
+WORKDIR /opt/kroxylicious
+COPY . .
+RUN ./mvnw clean verify -Pdist -Dquick
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.8-860
+
+ARG JAVA_VERSION=17
+ARG TARGETOS
+ARG TARGETARCH
+ARG KROXYLICIOUS_VERSION
+
+RUN microdnf update \
+    && microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y java-${JAVA_VERSION}-openjdk-headless-17.0.7.0.7 openssl-1.1.1k shadow-utils-4.6 \
```

Comment:
can we avoid hard coding the micro/build numbers in the java release so that we pickup from CVE respins without changes on our end?

Reply:
Yep, this was addressing lift warnings, I imagine I should also unpin the ubi-minimal version for the same reason and ignore those warnings.

Reply:
Unpinning leads to non-deterministic builds! I think CVE respins etc should be handled by passing in a new target version....

Reply:
We need to revisit this for building images for releases/production use. I'm ok with unpinned in the dev workflow.